packages/monaco-contributions/common.js

```diff
@@ -0,0 +1,30 @@
+export const EVALUATE_KEYWORD = [
+  "CTX",
+  "STATE",
+  "DATA",
+  "EVENT",
+  "FLAGS",
+  "APP",
+  "ANCHOR",
+  "HASH",
+  "INSTALLED_APPS",
+  "ITEM",
+  "LOCAL_STORYAGE",
```

Comment:
你的 IDE 需要启用下 spellcheck

```suggestion
  "LOCAL_STORAGE",
```

Reply:
😂